haystack/utils/docker.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 
-def cache_models(models=None):
+def cache_models(models=None, use_auth_token=None):
```

Comment:
We should add docstrings describing the parameters and type annotations here as well.